.env.example

```diff
@@ -9,5 +9,4 @@ TWILIO_AUTH_TOKEN=Your-Twilio-Auth-Token
 TWILIO_NUMBER=Your-Twilio-Phone-Number
 
 # Your mongo db urls
-MONGO_URL=Mongo-Url
```

Comment:
Why did you remove this env variable? Please take a look at https://github.com/TwilioDevEd/eta-notifications-node/blob/15d4900572e3fdc53043684d58aa8e0362eedd58/config.js#L27